Sources/Camera/ZLCustomCamera.swift

```diff
@@ -1308,6 +1310,7 @@ open class ZLCustomCamera: UIViewController {
     
     private func playRecordVideo(fileURL: URL) {
         recordVideoPlayerLayer?.isHidden = false
+        cameraConfig.overlayView?.isHidden = true
```

Comment:
Line 1352 in 
```swift
public func photoOutput(_ output: AVCapturePhotoOutput, didFinishProcessingPhoto photoSampleBuffer: CMSampleBuffer?, previewPhoto previewPhotoSampleBuffer: CMSampleBuffer?, resolvedSettings: AVCaptureResolvedPhotoSettings, bracketSettings: AVCaptureBracketedStillImageSettings?, error: Error?)
```
 method also needs this line of code.

Reply:
AFAIU photo preview is happening separately from the custom camera, so it is not affected by overlay, only video preview is. But I will also check this, thanks!

Reply:
Ah now I see, yes I forgot about this flag, you are right, also, in that case, `cameraConfig.overlayView?.isHidden = true` has to be moved upper from the `videoURL` block (done in the last commit).